javascript/javascript2/week1/index.js

```diff
@@ -58,7 +58,7 @@ function generateAnimalName(e) {
 // *********
 
 function getUserName(e) {
-  if (nameArea.value != "") {
+  if (nameArea.value != "" && typeof nameArea.value === "string") {
```

Comment:
Wanted to put condition that checking blank space, but didn't work out =(

Reply:
Try This `if (nameArea.value !== '' && nameArea.value.trim())`